tet.rs/src/tetris/board/piece/piece_matrix.rs

```diff
@@ -0,0 +1,136 @@
+pub type MatrixT = u16;
+
+use crate::tetris::board::piece::{
+    piece_position::PositionT, piece_rotation::PieceRotation, piece_type::PieceType,
+};
+
+const IN: u16 = 0b0000111100000000;
```

Comment:
Neat trick, you can add underscores in between digits which can help readability:
```suggestion
const IN: u16 = 0b0000_1111_0000_0000;
```

Reply:
Neat !